src/utils/save_all_recordings.py

```diff
@@ -4,20 +4,24 @@
 import pandas as pd
 
 
-def save_all_recordings(recordings: 'list[Recording]', path: str):
+def save_all_recordings(recordings: 'list[Recording]', folder_path: str, file_name: str) -> None:
     """
     Save all recordings to a single csv file.
+
+    Refactoring idea:
+    - check if there is a file already and delete it?
```

Comment:
This would be a nice issue, but won't be done if we leave it as a comment in the code.